interpreter/cling/lib/Utils/AST.cpp

```diff
@@ -49,6 +49,9 @@ namespace utils {
   NestedNameSpecifier* GetFullyQualifiedNameSpecifier(const ASTContext& Ctx,
                                                       NestedNameSpecifier* scope);
 
+  template<typename D>
+  static D* LookupResult2Decl(LookupResult& R);
```

Comment:
Can you remove this and move the definition into an unnamed namespace?

Reply:
Yes of course.